src/components/SettingsPane/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import {
+	Box,
+	Card,
+	CardContent,
+	CardHeader,
+	Stack,
+	Typography,
+} from '@mui/material';
+import { color } from '@balena/design-tokens';
+
+export interface SettingsPaneProps {
+	title?: string | React.ReactNode;
+	children: React.ReactNode;
+	danger?: boolean;
+}
+
+export interface SettingsPaneItemProps {
+	title: string | React.ReactNode;
+	description?: string | React.ReactNode;
+	action?: React.ReactNode;
```

Comment:
Do we have an instance where there is no action? This feels like it should be required

Reply:
I see the username example in the story (which I assume is because we have it in the account preferences page 🤔 

If we have no other such instances of no action, I think we can keep this *for now* but please leave a TODO that says we should make `action` not optional once usernames can be changed. We have had a GH issue for a bit now (see: https://github.com/balena-io/balena-ui/issues/1150) and I think Thodoris has told me it's technically possible and just needs to be implemented

OR you can make it required from the getgo (so we don't have to do a major later) and then give username a fake action (with a TODO of course)

Reply:
I'll go with optional and the TODO @myarmolinsky, if after we refactor all our settings page we end up with only those two cases (username and social account) then we can require it